app/adapters/internal/websocket/discord_bot.py

```
@@ -655,6 +672,388 @@ async def check_price(ctx: commands.Context[Any], market: str = "KRW-BTC") -> No
+            if target_profit_rate <= 0 or target_profit_rate > 1:
+                await ctx.send(
+                    f"{DiscordConstants.EMOJI_ERROR} 목표 수익률은 0보다 크고 100% 이하여야 합니다."
+                )
+                return
+
+            if (
+                price_drop_threshold >= 0
+                or price_drop_threshold < MIN_PRICE_DROP_THRESHOLD
+            ):
+                await ctx.send(
+                    f"{DiscordConstants.EMOJI_ERROR} 하락 기준은 0보다 작고 {abs(MIN_PRICE_DROP_THRESHOLD):.0%} 이상이어야 합니다."
```

Pylint (3.3.7)

[convention] Line too long (114/100)

(C0301)